ml3d/vis/boundingbox.py

```diff
@@ -245,7 +245,7 @@ def plot_rect3d_on_img(img,
         if color is None:
             color = np.ones((line_indices.shape[0], line_indices.shape[1], 3))
         for i in range(num_rects):
-            corners = rect_corners[i].astype(np.int)
+            corners = rect_corners[i].astype(np.int_)
```

Comment:
Hi @YishGene thanks for submitting this bugfix PR! Can we just use `int`? since `np.int is int` is `True` by definition. Also would appreciate it if you do a `git grep np.int` to find and fix any other places in the code with this issue.

Reply:
Thanks! Updating to int32 and merging!

Reply:
Thanks for taking care of this!